src/prog/main.F90

```diff
@@ -590,10 +593,18 @@ subroutine xtbMain(env, argParser)
       select type(xtb => calc%real_low)
       type is(TxTBCalculator)
          call chk%wfn%allocate(mol%n,xtb%basis%nshell,xtb%basis%nao)
+         call newWavefunction(env,mol,xtb,chk)
```

Comment:
Is it necessary here to call the `newWavefunction` subroutine every time?

Reply:
What do you mean by "every time"?